api/src/main/scala/uk/ac/wellcome/platform/api/Server.scala

```diff
@@ -17,27 +20,33 @@ import io.swagger.models.Swagger
 object ServerMain extends Server
 object ApiSwagger extends Swagger
 
+object ApiJacksonModule extends FinatraJacksonModule {
+  override val propertyNamingStrategy = CamelCasePropertyNamingStrategy
+}
+
 class Server extends HttpServer {
   override val name = "uk.ac.wellcome.platform.api Platformapi"
   override val modules = Seq(ElasticClientModule)
 
   private final val apiName = flag(name = "api.name", default = "catalogue", help = "API name path part")
   private final val apiVersion = flag(name = "api.version", default = "v0", help = "API version path part")
+  private final val apiPrefix = flag(name = "api.prefix", default = "/" + apiName() + "/" + apiVersion(), help = "API path prefix")
+
+  flag(name = "api.context", default =  apiPrefix() + "/context.json", help = "API JSON-LD context")
 
-  flag(name = "api.prefix", default = "/" + apiName() + "/" + apiVersion(), help = "API path prefix")
+  override def jacksonModule = ApiJacksonModule
 
   override def configureHttp(router: HttpRouter) {
     router
       .filter[CommonFilters]
       .filter[LoggingMDCFilter[Request, Response]]
       .filter[TraceIdMDCFilter[Request, Response]]
       .filter(CorsFilter())
-      .add[CollectionController]
+      .add[ContextController]
```

Comment:
nit: let’s put the `ContextController` towards the end – so our other Finatra apps looks semi-consistent: ManagementController, MainController, custom controllers.